config/prow/cluster/starter-s3.yaml

```diff
@@ -532,7 +532,7 @@ spec:
   - port: 80
     targetPort: 8888
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
I suspect this will break kubectl apply, right?

Reply:
Do you mean because there might be `extensions/v1beta1` version still available on the cluster?

Reply:
Oh sorry these are both just starter examples? Nevermind.
/hold cancel

Reply:
(my concern is that changing the api version of our production ingress resources will cause our production deployments to start failing as I would expect that field to be immutable (could be wrong, but not something I want to merge until the morning)

Reply:
Yes I agree, I was hesitating too to update it, but went ahead considering these are starters for someone like me who is trying to setup a prow cluster.

Reply:
Totally, thanks so much for cleaning this up!!